### DISCUSSION TITLE
CTT When Browse Direction is Invalid i get BadInternalError

### DISCUSSION THREAD

**Discussion by johannwesely** [eclipse-milo/milo#1312] [General]
Hi in View Basic i also have this issue:
![image](https://github.com/user-attachments/assets/ae72a4cc-b6c6-4cc8-a71a-a832459fd7b5)
I tried to Browse from RootNode Inverse with milo client and get Status Code Good is this normal behaviour?
`
```
BrowseDescription browse = new BrowseDescription(NodeIds.RootFolder, BrowseDirection.Inverse, NodeIds.References, true, uint(NodeClass.Object.getValue() | NodeClass.Variable.getValue()), uint(BrowseResultMask.All.getValue()));

BrowseResult browseResult = client.browse(browse);
````
thanks in advance
Johann

**Comment by kevinherron** [comment]
Easy fix on this one, #1313 and #1314.
